pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -222,7 +237,8 @@ stdenv.mkDerivation rec {
     if [ -x $out/bin/qemu-system-${stdenv.hostPlatform.qemuArch} ]; then
       makeWrapper $out/bin/qemu-system-${stdenv.hostPlatform.qemuArch} \
                   $out/bin/qemu-kvm \
-                  --add-flags "\$([ -e /dev/kvm ] && echo -enable-kvm)"
+                  --run ${emit-kvm-warnings}/bin/emit-kvm-warnings \
```

Comment:
Might it be better to have this file inside the QEMU derivation?  You could use `passAsFile` and then just copy it into $out/libexec or something.

Reply:
done in 88899a82dac3c78805b28c490259f1a4c3dc82f8